api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -35,9 +40,31 @@
 )
 CensusDirectory = Dict[CensusVersionName, Union[CensusVersionName, CensusVersionDescription]]
 
+CensusMirrorName = str  # name of the mirror
+CensusMirror = TypedDict(
+    "CensusMirror",
+    {
+        "provider": str,  # provider of the mirror. Only S3 is supported in this version.
+        "base_uri": str,  # name of the bucket or resource
+        "region": Optional[str],  # region of the bucket or resource
```

Comment:
This comment is different than the comment for the same field in the Locator. If it has different semantics, I would document them, or rationalize the comments.

Reply:
The semantics are slightly different since `s3_region` is effectively a deprecated field (it will only be used by older versions of the census that do not have mirror resolution). I'll mark it explicitly.